src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -56,6 +57,24 @@ export const availableActions: AvailableActionsType = {
     },
 };
 
+export const volunteerAvailableActions: Omit<AvailableActionsType, "Delete Parcel"> = {
+    "Download Day Overview": {
+        newStatus: "Day Overview Downloaded",
```

Comment:
Is there a way we can avoid repetition here -> am just a little worried that if new actions are added, and new roles have access to different actions, then we could end having issues. e.g copy paste errors meaning that newStatus is not synced between different roles. 
Lmk know if you disagree with this / couldn't see a way to do this.

Reply:
I've added a new "availableToVolunteers" to the list of available actions and the list gets filtered in an useEffect. Please let me know if there is a better approach 👍